packages/components/src/button/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import { globalStyles } from "../index.styled";
```

Comment:
Can we change these to use the `@...` import alias?

Reply:
@Darianlmj do you mean the @csesoc/ui-components import?